students/minjeong/requirements.txt

```
@@ -0,0 +1,4 @@
+ango==3.2.4
+django-cors-headers==3.7.0
+mysqlclient==2.0.3
+PyMySQL==1.0.2
```

m1을 사용중이면 PyMySQL을 그렇지 않으면 mysqlclient만 사용해주세요!

```suggestion
PyMySQL==1.0.2
```

---

수정했습니다.